Source/WebCore/html/HTMLMediaElement.cpp

```diff
@@ -8392,7 +8392,6 @@ MediaProducerMediaStateFlags HTMLMediaElement::mediaState() const
     MediaStateFlags state;
 
     bool hasActiveVideo = isVideo() && hasVideo();
-    bool hasAudio = this->hasAudio();
```

Comment:
I think having these bool here would be easier to read more.
So, let's suppress the warning with `UNUSED_VARIABLE(hasAudio)` in the following `#if`'s `#else` case.